Lib/logging/__init__.py

```diff
@@ -916,8 +916,7 @@ def getHandlerNames():
     """
     Return all known handler names as an immutable set.
     """
-    result = set(_handlers.keys())
-    return frozenset(result)
+    return frozenset(_handlers.keys())
```

Comment:
Why keep `.keys()`?

Reply:
I've tested with `frozenset(_handlers)` it is a bit slower:

```
(.venv) ~/Desktop/cpython  issue-107710 ✗                                              1 ⚠️
» pyperf timeit --setup 'import ex; l = ex.logging' 'l.getHandlerNames()'
.....................
Mean +- std dev: 2.07 us +- 0.02 us
                                                                                           
(.venv) ~/Desktop/cpython  issue-107710 ✗                                                 
» pyperf timeit --setup 'import ex; l = ex.logging' 'l.getHandlerNames()'
.....................
Mean +- std dev: 2.07 us +- 0.01 us
                                                                                           
(.venv) ~/Desktop/cpython  issue-107710 ✗                                                 
» pyperf timeit --setup 'import ex; l = ex.logging' 'l.getHandlerNames()'
.....................
Mean +- std dev: 2.07 us +- 0.01 us
```

Reply:
I don't think a 3us difference is a good enough reason to avoid using the simpler code here. And it is still faster than `main` :)

Reply:
The difference is so small that I would classify this change as a code cleanup rather than an optimization.